websites/S/Stake/presence.ts

```diff
@@ -0,0 +1,80 @@
+const presence = new Presence({
+		clientId: "1030599970551771266",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const [showBal, showTimestamp, showCurrentGame] = await Promise.all([
+			presence.getSetting<boolean>("showBal"),
+			presence.getSetting<boolean>("showTimestamp"),
+			presence.getSetting<boolean>("showCurrentGame"),
+		]),
+		presenceData: PresenceData = {
+			largeImageKey: "logo",
```

Comment:
Imgur links are preferred for `largeImageKey` for easier maintainability.
```suggestion
			largeImageKey: "https://i.imgur.com/D4d2JSH.png",
```
Also, it looks like the logo from your metadata is different from the one in your screenshot. You may want to choose one of those images and use it here and in the metadata.